client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -444,126 +379,145 @@ class CampaignAdminList extends Component {
           {body}
         </div>
         <div className="toolbar toolbar--south">
-          {this.renderButtonToolbar()}
+          {renderButtonToolbar()}
         </div>
       </div>
     );
-  }
+  };
+
+  useEffect(() => {
+    setBreadcrumbs();
+    // Only load record if not already present
+    if (!loading) {
```

Comment:
Depending on discussion in another comment thread about why this isn't in state anymore, this might not be an appropriate change - it used to be specifically `if (!this.isRecordLoaded()) {`